File: SourceAFIS/General/Neighborhood.cs
using System;
using System.Collections.Generic;
using System.Text;
#if !COMPACT_FRAMEWORK
using System.Drawing;
#endif
using SourceAFIS.Dummy;

namespace SourceAFIS.General
{
    public static class Neighborhood
    {
        public static readonly Point[] EdgeNeighbors = new Point[] {
            new Point(0, -1),
            new Point(-1, 0),
            new Point(1, 0),
            new Point(0, 1)
        };

        public static readonly Point[] CornerNeighbors = new Point[] {
            new Point(-1, -1),
            new Point(0, -1),
            new Point(1, -1),
            new Point(-1, 0),
            new Point(1, 0),
            new Point(-1, 1),
            new Point(0, 1),
            new Point(1, 1)
        };
    }
}

File: java/sourceafis/src/sourceafis/general/Neighborhood.java
/**
 * @author Veaceslav Dubenco
 * @since 10.10.2012
 */
package sourceafis.general;

/**
 * 
 */
public class Neighborhood {
	public static final Point[] EdgeNeighbors = new Point[] { new Point(0, -1),
			new Point(-1, 0), new Point(1, 0), new Point(0, 1) };

	public static final Point[] CornerNeighbors = new Point[] {
			new Point(-1, -1), new Point(0, -1), new Point(1, -1),
			new Point(-1, 0), new Point(1, 0), new Point(-1, 1),
			new Point(0, 1), new Point(1, 1) };

}
